fictadvisor-web/src/app/admin/common/constants/index.ts

```diff
@@ -1,8 +1,10 @@
 export const useQueryAdminOptions = {
   staleTime: 1500,
-  cacheTime: 1500,
+  gcTime: 1500,
   refetchOnMount: false,
   refetchOnReconnect: false,
   refetchOnWindowFocus: false,
-  keepPreviousData: true,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
why here a eslint disable

Reply:
because i cannot find type for this args

Reply:
remove that property completely, as we actually dont really need it